source/applications/advanced/downsample.py

```diff
@@ -114,14 +104,14 @@ def _main():
     frame = zivid.Frame(filename_zdf)
 
     # Getting the point cloud
-    point_cloud = frame.get_point_cloud().to_array()
+    point_cloud = frame.point_cloud().copy_data("xyzrgba")
     xyz = np.dstack([point_cloud["x"], point_cloud["y"], point_cloud["z"]])
     rgb = np.dstack([point_cloud["r"], point_cloud["g"], point_cloud["b"]])
```

Comment:
May I suggest using copy_data("xyz") and copy_data("rgba") separately?

Reply:
I will make a new PR for downsample and use the SDK

Reply:
Ah yes, good point. I should have thought of that :)